server/models/Notification.js

```diff
@@ -40,6 +42,61 @@ export default function(Sequelize, DataTypes) {
     classMethods: {
       createMany: (notifications, defaultValues) => {
         return Promise.map(notifications, u => Notification.create(_.defaults({},u,defaultValues))).catch(console.error);
+      },
+
+      /**
+       * Get the list of subscribers to a mailing list (e.g. backers@:collectiveSlug.opencollective.com)
+       * For members, backers, and info: opt-in: we look for rows in the Notifications table
+       * For events, it's opt-out. We exclude people who have explicitly unsubscribed
+       */
+      getSubscribers: (collectiveSlug, mailinglist) => {
```

Comment:
This whole function and sub functions took me a while to parse. Would it not be easier to separate into two separate calls? One for events and one for mailing lists?